pyani_plus/methods/method_anim.py

```diff
@@ -76,7 +76,9 @@ def get_aligned_bases_count(aligned_regions: dict) -> int:
     return aligned_bases
 
 
-def parse_delta(filename: Path) -> tuple[int, int, float, int]:
+def parse_delta(
+    filename: Path,
+) -> tuple[int | None, int | None, float | None, int | None]:
```

Comment:
Can you say ``tuple[int,int,float,int] | tuple[None,None,None,None]`` which is narrower?